web/src/components/njwds-extended/Tag.tsx

```diff
@@ -70,16 +73,7 @@ export const Tag = (props: Props): ReactElement => {
   const disableUppercase = props.isLowerCase ? "text-no-uppercase" : "";
   const radius = props.isRadiusMd ? "radius-md" : "";
 
-  const className = [
-    defaultStyle,
-    styling,
-    hoverStyling,
-    textWrap,
-    fixedWidth,
-    disableUppercase,
-    radius,
-    props.className,
-  ]
+  const className = [defaultStyle, styling, hoverStyling, textWrap, fixedWidth, disableUppercase, radius]
     .map((i) => {
       return i?.trim();
     })
```

Comment:
[sand] This is probably code best handled with `.reduce()`. Also I believe the optional on line 78 is no longer required now that `props.className` is no longer included in the list.